src/NUnitConsole/nunit3-console/CommandLineOptions.cs

```diff
@@ -402,7 +400,7 @@ private void ResolveOutputSpecification(string value, IList<OutputSpecification>
 
             if (spec.Transform != null)
             {
-                var transformPath = Path.Combine(WorkDirectory, spec.Transform);
+                var transformPath = Path.Combine(Environment.CurrentDirectory, spec.Transform);
```

Comment:
Same concept applies here. We have to be sure that `Environment.CurrentDirectory` is not changed before we execute this. Since NUnit should __never__ change it, that essentially means that no user code can be allowed to execute before evaluating this line. Odds are, we are safe but "belt and suspenders" may apply.